src/index.html

```diff
@@ -10,7 +10,58 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label
+        for="bigSearch"
+        class="search__label"
+      >
+        <img
+          src="images/Search.svg"
+          alt="search icon"
+          class="big__icon"
+        >
+      </label>
+      <input
+        class="search big__search"
+        type="search"
+        name="bigSearch"
+        id="bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="small"
+      class="form"
+    >
+      <label
+        for="smallSearch"
+        class="search__label"
+      >
+        <img
+          src="images/Search.svg"
+          alt="search icon"
+          class="small__icon"
```

Comment:
then like above this class should be following to the BEM convention `icon icon--small` and styling from `small__icon` should moved to `icon--small` modifier class